### DISCUSSION TITLE
✨ Enable option for SSH key and GPG key archivation

### DISCUSSION THREAD

**Discussion by sitole** [community/community#12884] [Other Feature Feedback, Questions, & Ideas]
Hello!
Right now, i setup new GPG key for my GitHub account and i dont want to use old one.
But right now i cannot delete this GPG key because all commits will be marked as without signature.
So, it would be good to provide some possibility how to "archive" old SSH/GPG keys. It means commits will stay signed but its not possible to use this key in future. ❤️

**Comment by stijnbollen** [comment]
I know this one is older, but the discussion at https://github.com/orgs/community/discussions/14008 has better visibility, so we should focus on that one.
